root/static/js/canto-modules.js

```diff
@@ -1296,11 +1296,17 @@ function featureChooserControlHelper($scope, $uibModal, CursGeneList,
   if ($scope.featureType === 'gene') {
     getGenesFromServer();
   } else {
-    CursGenotypeList.cursGenotypeList().then(function (results) {
-      $scope.features = results;
-    }).catch(function () {
-      toaster.pop('note', "couldn't read the genotype list from the server");
-    });
+    if ($scope.featureType === 'genotype') {
+      CursGenotypeList.cursGenotypeList().then(function (results) {
+        $scope.features = results;
+      }).catch(function () {
+        toaster.pop('note', "couldn't read the genotype list from the server");
+      });
+    } else {
+      Curs.list('metagenotype').then(function (res) {
+        $scope.features = res.data;
+      });
```

Comment:
I presume there needs to be a catch block after `Curs.list('metagenotype')`, like the call to `cursGenotypeList()` above?

Reply:
Thanks.  I've added a .catch()